src/Selenium2Library/keywords/_browsermanagement.py

```diff
@@ -617,3 +644,39 @@ def _parse_capabilities_string(self, capabilities_string):
             desired_capabilities[key.strip()] = value.strip()
 
         return desired_capabilities
+
+
+class CapabilitiesConfig(object):
+
+    def __init__(self, arg):
+        for k in arg:
+            self.__setattr__(k, arg[k])
```

Comment:
I already proposed an alternative to use `self.__dict__.update(capabilities)`. As I already commented the PR itself, it might be better to return a custom dict instead.